functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectConfigurator.java

```diff
@@ -103,9 +113,15 @@ public Project enable(Using usingInstallation) {
                     .build();
             }
 
-            this.project.configureSmartTesting()
-                .withConfiguration(configuration)
-                .createConfigFile();
+            if (customConfigFile != null) {
+                this.project.configureSmartTesting()
+                    .withConfiguration(configuration)
+                    .createConfigFile(customConfigFile);
+            } else {
+                this.project.configureSmartTesting()
+                    .withConfiguration(configuration)
+                    .createConfigFile();
+            }
```

Comment:
Btw, if you moved the lines
[Paths.get(root.toString(), "smart-testing.yml");](https://github.com/arquillian/smart-testing/pull/254/files#diff-7aaa07c0a9bc90f3bf91fc1b16ce0fa3R134) 
[Paths.get(root.toString(), customConfigFile)](https://github.com/arquillian/smart-testing/pull/254/files#diff-7aaa07c0a9bc90f3bf91fc1b16ce0fa3R139)
here to this if, then you could use the same method `createConfigFile(customConfigFile)` for both cases